src/functions.php

```diff
@@ -164,3 +164,54 @@ function getVersion()
 {
     return 'v2.6.0';
 }
+
+/**
+ * @param $array
+ * @param $attribute
+ * @param $currentValue
```

Comment:
Missing types for arguments in the PHPDoc. Also, I don’t think this function should be able to return false.